src/index.js

```diff
@@ -1,10 +1,35 @@
 'use strict';
 /**
  * @module braintree-web-drop-in
+ * @description There are two ways to integrate Drop-in into your page. You can use [`dropin.create`](#.create) or a standalone script tag integration (example below).
```

Comment:
Is this where this belongs? Should it be mentioned on the home page of the docs as well?